scripts/_helpers.py

```diff
@@ -228,7 +228,11 @@ def load_network(import_name=None, custom_components=None):
     pypsa.Network
     """
     import pypsa
-    from pypsa.descriptors import Dict
+
+    try:
+        from pypsa.descriptors import Dict
+    except:
+        from pypsa.definitions.structures import Dict  # from pypsa version v0.31
```

Comment:
A great catch for now, and I think ready to go as a part of this PR.

A small consideration for future: may we be a bit more specific here to avoid getting into weird troubles? PyPSA is currently in process of structure revisions, and it would be great to account for that. Can we probably check for the method existence directly, like [here](https://stackoverflow.com/a/54625079/8465924)?

Reply:
Honestly, I don't think it is worth over optimizing these blocks too much.
When we update the environment, we may likely need to specify a higher pypsa version and we can drop this ugly (I agree) try catch. Moreover, there can be more surprises we will need to tackle and time can be better spent there

Reply:
if you don't like this try catch, I only keep the <0.31 version. I left it here as I had it in the past attempts and it would be good to have it at hand when the problem will arise

Reply:
I completely support the idea to increase the version as much as we can 🙂 You put a lot of effort making it work.